portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,7 +27,7 @@
 public class DataServlet extends HttpServlet {
 
   
-  private List<String> quotes;
+  private ArrayList<String> quotes;
```

Comment:
Better to leave this as List. 

You can also make this 
private final List<String> quotes = new ArrayList<String>(). And remove the new from the init method.

Reply:
Done